Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,46 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '(@){{plainid}}\.'
```

Comment:
done.

@FichteFoll I kept "punctuation.accessor.scala" in order to stay coherent with the rest of the file. This scope renaming could go in a separate PR.